.travis.yml

```diff
@@ -33,29 +34,31 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
+    - php: '7.1'
       env: SYMFONY=2.8.*
-    - php: '7.0'
+    - php: '7.1'
+      env: SYMFONY=3.0.*
```

Comment:
[This dev-kit PR](https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/pull/415/files) changed all zeroes into ones…